src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -288,6 +290,7 @@ private IEnumerable<MountPointInfo> FindMountPointsToScan(bool forceRebuild)
 
                 if (forceScanAll)
                 {
+                    returnedPoints.Add(template.ConfigMountPointId);
```

Comment:
Not yieldnig more then once is goal of this change, for example when running unit test we returned 150 MountPoints instead of 2, which resulted in 10s instead 1 sec execution re-scanning of mountpoint for same templates over and over again.